Cargo.toml

```diff
@@ -21,5 +21,6 @@ qstring = "0.7.2"
 solana-sdk = "=1.14.17"
 spacemesh-derivation-path = { path = "derivation-path", version = "=0.0.1" }
 spacemesh-remote-wallet = { path = "remote-wallet", version = "=0.0.1" }
+spacemesh-sdkutils = { path = "sdkutils", version = "=0.0.1" }
 thiserror = "1.0.40"
 uriparse = "0.6.4"
```

Comment:
To fix the rpath issue on mach this file I think needs the following configuration:

```Cargo.toml
[profile.release-clib]
inherits = "release"
strip = true
lto = true
rpath = true
```

Reply:
https://github.com/spacemeshos/smkeys/issues/6#issuecomment-1563099029

Reply:
I'm going to go ahead and merge without this. If it's still causing issues we can open a new PR to fix it (as you did with #5)

Reply:
I'm not sure if related but I created a new issue: https://github.com/spacemeshos/spacemesh-sdk/issues/12